### PR TITLE
Bug Fix : Enabled Filter by C#, Java & C++

### PR DESCRIPTION
**Pull request by sanjay-09**
1. As In this repository, we are dealing with the special characters used in the languages like C#,C++.Hence we were not able 
to fetch the langfilter correctly when the langFilter contains the special character.
url: `/api/project?lang=${langFilter}&sort_by=${sortFilter.value}&order=${sortFilter.order}`.

2. So I have encoded the langFilter before fetching the data using the encodeURIComponent() function,
     
image

3.Then I have used the filter statement to filter out the records which are coming from the mongoDb,because for an language C we are getting all the records where C is the prefix so I have filtered those records.
image

4.For the language C++, As it contains the extra special Character '++', its regex was not matching while quering the database. So for this I have created a manual regex to match the records where the language is 'C++'
image

I have tested out all the cases and getting the records as per the expectation.
image
image

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GITHUB_TOKEN=
 # (In hours)
 REPO_CACHE_TIME=8
-MONGODB_URI=
+MONGODB_URI=mongodb+srv://vdnh123:abc123456@atlascluster.4qkviv9.mongodb.net/firstIssue
```
Comment (aadeshk-beam) [inline .env.local:4]: Please remove this configuration

Comment (aadeshkulkarni) [inline .env.local:4]: @sanjay-09 - .env shouldn't contain secrets. Please remove your db connection string.

**Review by aadeshkulkarni — CHANGES_REQUESTED** [review]
Please fix the PR comment. Everything looks fine.

**Comment by sanjay-09** [comment]
Updated the PR Comment.

**Review by aadeshkulkarni — APPROVED** [review]

**Comment by aadeshkulkarni** [comment]
Thanks for your contribution!

**Pull request merged**